scripts/setup_system_rbpi_raspbian_lite_buster.sh

```diff
@@ -260,6 +260,10 @@ systemctl enable a2jmidid
 systemctl enable zynthian
 systemctl enable zynthian-webconf
 systemctl enable zynthian-config-on-boot
+#if we have an EPDF hat, enable the pwm fan service
```

Comment:
This script is only executed for building a SD image (i.e. nightly build system). It's normally executed on an emulated environment, not a true zynthian or RBPi. I think it has no sense to try to detect specific hardware on this stage ...

Reply:
Ok, I was looking for where the zynthian services were enabled so I could get the fan service installed as well. I'll look for a better place to detect and enable the pwm fan service.